### PR TITLE
Refactored simulation model to secondary model

### PR DESCRIPTION
**Pull request by favreau**
Geometry using the SECONDARY_MODEL_MATERIAL_ID are added to the secondary model.

### CODE REVIEW

File: engines/ospray/OSPRayModel.cpp [hunk 1]
```diff
@@ -182,10 +183,17 @@ void OSPRayModel::_commitSpheres(const size_t materialId)
 
     ospCommit(geometry);
 
-    if (materialId == BOUNDINGBOX_MATERIAL_ID)
+    switch (materialId)
```
Comment (tribal-tec) [inline engines/ospray/OSPRayModel.cpp]: this block shall be a function as it repeats some times

**Comment by tribal-tec** [comment]
commit & PR message are wrong?

**Comment by hernando** [comment]
I find this commit a bit strange, can you give more background on what are you trying to achieve? Maybe there are other ways

**Comment by hernando** [comment]
The commit message is also incorrect because you're not refactoring anything, you're adding stuff as the simulation model was gone.

**Pull request merged**